<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timesheet Entry</string>
  </property>
  <property name="windowFilePath">
   <string>window file path</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="whatsThis">
         <string/>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Timesheet</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Main title</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Full Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="fullname">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>10</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>230</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="statusTip">
                 <string>Enter full name as First Last</string>
                </property>
                <property name="inputMethodHints">
                 <set>Qt::ImhPreferUppercase|Qt::ImhUppercaseOnly</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="placeholderText">
                 <string>Enter full name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="employee_id">
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="statusTip">
                 <string>employee ID</string>
                </property>
                <property name="inputMethodHints">
                 <set>Qt::ImhDigitsOnly</set>
                </property>
                <property name="maxLength">
                 <number>5</number>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="editNameButton">
                <property name="statusTip">
                 <string>Edit name</string>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Date (week ending)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="date_field">
                <property name="toolTip">
                 <string>Enter the Saturday date for end of week</string>
                </property>
                <property name="statusTip">
                 <string>This is a status tip</string>
                </property>
                <property name="displayFormat">
                 <string>yyyy-MM-dd</string>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
                <property name="date">
                 <date>
                  <year>2020</year>
                  <month>1</month>
                  <day>1</day>
                 </date>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>6</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="rowCount">
               <number>5</number>
              </property>
              <property name="columnCount">
               <number>9</number>
              </property>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelThisWeekGoals">
              <property name="text">
               <string>Goals from last week</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="goals1">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
              <property name="documentTitle">
               <string>doc title</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::TextEditorInteraction</set>
              </property>
              <property name="placeholderText">
               <string>this is what I did this week</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelFutureGoals">
              <property name="text">
               <string>Future goals</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="goals2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
              <property name="documentTitle">
               <string>doc title</string>
              </property>
              <property name="placeholderText">
               <string>Future goals</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Reports</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="OK">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>employee_id</tabstop>
  <tabstop>date_field</tabstop>
  <tabstop>goals1</tabstop>
  <tabstop>goals2</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>editNameButton</tabstop>
  <tabstop>OK</tabstop>
  <tabstop>Cancel</tabstop>
  <tabstop>fullname</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
